actionpack/lib/action_dispatch/middleware/debug_view.rb

```diff
@@ -2,8 +2,6 @@
 
 # :markup: markdown
 
-require "pp"
```

Comment:
Did you remove this yourself, or was that RuboCop? RuboCop should have kept this since `PP` is referenced later on in the file.